boot-mongodb-elasticsearch/src/main/java/com/example/mongoes/web/controller/RestaurantController.java

```diff
@@ -26,60 +27,60 @@
 @RestController
 @Timed
 @RequestMapping("/api/restaurant")
-public class RestaurantController {
+@Validated
+class RestaurantController {
 
     private final RestaurantService restaurantService;
 
-    public RestaurantController(RestaurantService restaurantService) {
+    RestaurantController(RestaurantService restaurantService) {
         this.restaurantService = restaurantService;
     }
 
     @GetMapping
-    public Mono<ResponseEntity<SearchPage<Restaurant>>> findAllRestaurants(
+    Mono<ResponseEntity<SearchPage<Restaurant>>> findAllRestaurants(
```

Comment:
@coderabbitai, when called from swagger this endpoint it is returning below error, please fix it

"path": "/api/restaurant",
  "status": 500,
  "error": "Internal Server Error",
  "requestId": "6cacd6f4-24",
  "message": "HV000030: No validator could be found for constraint 'jakarta.validation.constraints.Size' validating type 'java.lang.Integer'. Check configuration for 'findAllRestaurants.limit'",